src/network/pages/SponsorshipsPage.tsx

```diff
@@ -63,7 +63,7 @@ export const SponsorshipsPage = () => {
             : mySponsorshipsQuery
 
     const sponsorships: SponsorshipElement[] =
-        (sponsorshipsQuery?.data?.pages.flat() as SponsorshipElement[]) || []
+        sponsorshipsQuery?.data?.pages.map((page) => page.elements).flat() || []
```

Comment:
`sponsorshipsQuery` isn't nullish.

```suggestion
        sponsorshipsQuery.data?.pages.map((page) => page.elements).flat() || []
```